apps/codeforafrica/src/lib/api.ghost/index.js

```diff
@@ -47,3 +47,10 @@ export async function getRelatedStories(story) {
       s.tags.some((t) => story.tags.find((st) => equalsIgnoreCase(t, st)))
   );
 }
+
+export async function getStoriesRelatedToProject(projectName) {
+  const stories = await getAllStories();
+  return stories.filter((s) =>
+    s.tags.some((t) => equalsIgnoreCase(t, projectName))
+  );
+}
```

Comment:
I think the two functions `getRelatedStories` and `getStoriesRelatedToProject` can be combined into `getRelatedStoriesByTags(...tags)` where tags is an array of strings.

For stories, we would then call this function `getRelatedStoriesByTags(story.tags)` and for projects, we could call it as `getRelatedStoriesByTags([project.name])`... or something like that.